snooty/rstspec.toml

```diff
@@ -899,6 +899,10 @@ example = """.. topic:: ${1:Title of block}
    ${0:Content}
 """
 
+[directive.search-results]
+help = """Shows search results from browser query params (no arguments)."""
+argument_type = "none"
```

Comment:
Just exclude this line -- this is a string called "none", which technically in the current implementation asks for an optional argument.